### PR TITLE
Changed error message of deploy apps to be more descriptive

### LINKED ISSUE #685: Improve error messaging on apps deploy.
Currently, if you run a `deploy-apps` command, and the environment config is missing in the `.env.json` file, the output looks like this:

image

Ok, it does tell you that the environment config is missing. But it **doesn't** tell you that you get that config by deploying an API for that respective environment, i.e.: `webiny deploy-api --env=dev`. 

We need to improve the message and output something like:
```
Environment "dev" is missing in ./apps/admin/.env.json file.
Have you deployed the corresponding API environment?
To deploy the API, run "webiny deploy-api --env=dev"
```

The related code is located here: `packages/cli/sls/deploy.js:43`

@doitadrian how does this look to you?

### PR DESCRIPTION
**Pull request by ahmad-reza619**
## Related Issue

close #685 

## Your solution

I have changed the error message of `webiny deploy-apps` to be more descriptive so that it's a little bit more descriptive

## How Has This Been Tested?

1. run `webiny deploy-apps --env=dev` with prerequisites of not having  dev environment .env.json files in apps
2. there will be an error saying
```
Environment {name of environment} is missing {app path}/.env.json file.
Have you deployed the corresponding API environment?
To deploy the API, run "webiny deploy-api --env=dev."
```
## Screenshots (if relevant):

### CODE REVIEW

File: packages/cli/sls/deploy.js [hunk 1]
```diff
@@ -47,9 +47,13 @@ module.exports = async inputs => {
...
             if (!envJson[env]) {
                 console.log(
-                    `🚨 Environment ${green(env)} is missing in ${green(
-                        app.name
-                    )} app. Check your ${green(app.location + "/.env.json")} file.`
+                    [
+                        `🚨 Environment ${green(env)} is missing in ${green(
+                            app.location + "/.env.json"
+                        )} file.`,
+                        "Have you deployed the corresponding API environment?.",
```
Comment (Pavel910) [inline packages/cli/sls/deploy.js]: Please remove the `.` after `?`

File: package.json [hunk 2]
```diff
@@ -141,5 +141,8 @@
       "cliPath": "node_modules/commitizen",
       "path": "./scripts/cz-adapter"
     }
+  },
+  "dependencies": {
+    "node-sass": "^4.13.1"
```
Comment (Pavel910) [inline package.json]: Could you please remove this dependency? I know it's the one that was causing you problems at the beginning but I think it's not necessary here.

File: packages/cli/sls/deploy.js [hunk 3]
```diff
@@ -47,9 +47,13 @@ module.exports = async inputs => {
...
+                        'To deploy the API, run "webiny deploy-api --env=dev".'
```
Comment (Pavel910) [inline packages/cli/sls/deploy.js]: For better readability, it is best to colorize the text in quotes, so please apply the `green` or `blue` color to the quoted text (and remove the quotes).

File: package.json (hunk 2 above)
Comment (ahmad-reza619) [reply package.json]: will do 😄

File: packages/cli/sls/deploy.js (hunk 1 above)
Comment (ahmad-reza619) [reply packages/cli/sls/deploy.js]: sorry mistyped that one 😅

File: packages/cli/sls/deploy.js [hunk 4]
```diff
@@ -51,8 +51,8 @@ module.exports = async inputs => {
                         `🚨 Environment ${green(env)} is missing in ${green(
                             app.location + "/.env.json"
                         )} file.`,
-                        "Have you deployed the corresponding API environment?.",
-                        'To deploy the API, run "webiny deploy-api --env=dev".'
+                        "Have you deployed the corresponding API environment?",
+                        `To deploy the API, run ${blue('webiny deploy-api --env=dev')}.`
```
Comment (Pavel910) [inline packages/cli/sls/deploy.js]: This message should actually include the working command, so user can C/P it and run, so please make it `--env=${env}` so we get an actual environment that is missing in the command. Thanks!

Comment (ahmad-reza619) [reply packages/cli/sls/deploy.js]: oh yes, i forgot that the env is dynamic, sorry 😅

**Comment by Pavel910** [comment]
Looks good! Thank you for your time and congratulations on your first PR! 🎉

**Comment by ahmad-reza619** [comment]
> Looks good! Thank you for your time and congratulations on your first PR! 🎉

Sweet! Thank you too for all of your help 😄 glad i can be help

**Pull request merged**